tests/Unit/CacheInvalidatorTest.php

```diff
@@ -91,7 +90,7 @@ public function testInvalidatePath()
 
     public function testRefreshPath()
     {
-        /** @var MockInterface|RefreshInterface $refresh */
+        /* @var MockInterface|RefreshInterface $refresh */
```

Comment:
this change is wrong. The right fix is to move the inline phpdoc one line below, to be just before the variable declaration